seq, queue, and iteration columns

fartwhif opened the pull request:
added option for ACE style packet header flag nomenclature

fartwhif committed:
added seq, queue, and iteration columns to main list
added option for ACE style packet header flag nomenclature

fartwhif commented:
It could very well be setting the column values to incorrect values.  I would greatly appreciate it if anyone who knows more about queue could please verify the changes.

Slushnas commented:
The code looks good. There is an issue with the main form size and I have a couple other thoughts:

1. Some form element size settings in aclogview/Form1.Designer.cs got overwritten and result in the main form not looking correct (it is scrunched up). In my experience this can happen when making changes in the designer. The fix I've used is to go into that file, look at the diff, and revert the unwanted changes.
2. You have to reload the file or restart the program in order to see the ACE packet headers because they are part of the initial pcap processing. I would maybe change the option from "ACE style packet header flags" to "ACE style packet header flags (requires reload of file or program restart)" in order to let the user know what to expect.
3. It may be helpful to add a modified version of the GameMessageGroup enum from ACE that provides the name of the queue to display it in the main column instead of the number. Here is what I am thinking:

```
public enum MessageQueue
{
    Invalid        = 0x00,
    Event          = 0x01,
    Control        = 0x02,
    Weenie         = 0x03,
    Login          = 0x04,
    Database       = 0x05,
    SecureControl  = 0x06,
    SecureWeenie   = 0x07, // Autonomous Position
    SecureLogin    = 0x08,
    UIQueue        = 0x09,
    Smartbox       = 0x0A,
    Observer       = 0x0B,
    QueueMax       = 0x0C
}
```

fartwhif committed:
modified comments

fartwhif committed:
Merge remote-tracking branch 'remotes/upstream/ACE' into stuff2

fartwhif committed:
resizing groupbox for listview options of settings form
adding lookup enum for MessageQueue column

Slushnas committed:
Widen packet sequence column.

Slushnas committed:
Fix exception when sorting on the Queue column.
Add wait cursor to column sorting.

fartwhif committed:
Fix exception when sorting on the Queue column.

Fix exception when sorting on the Queue column. 
Add wait cursor to column sorting.

fartwhif committed:
updating change log
